frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -129,10 +129,7 @@ export const EmbedFrame = ({
       setHasInnerScroll(document.documentElement.scrollTop > 0);
     };
 
-    document.addEventListener("scroll", handleScroll, {
-      capture: false,
```

Comment:
`capture: false` [is a default value ](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#capture). Having it here feels like the default value is otherwise.